packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -186,6 +199,46 @@ export class InteractionTracker {
     });
   }
 
+  private interceptXHRTasks(task: AsyncTask) {
+    if (isTrackedTask(task) && task.target instanceof XMLHttpRequest) {
+      const rootSpan: RootSpan = task.zone.get('data').rootSpan;
```

Comment:
What would you think about making both branches of this `if` statement into helper functions?

Reply:
Done